tests/src/assets/test_kvp.yaml

```diff
@@ -28,4 +28,23 @@ decodeTest:
   checkFloat: 5.1
   checkDoublePi: 3.14159265358979323846
   checkDoubleScientific : -4.2e8
-  checkDoubleInvalid : invalid_string
\ No newline at end of file
+  checkDoubleInvalid : invalid_string
+  checkBytesSpace: 0x00 0xff 0xff 0xff 0xff 0xff 0xff 0x00 0x15 0x85 0x1b 0x2a 0x01
+    0x01 0x01 0x01 0x00 0xff 0xff 0xff 0xff 0xff 0xff 0x00 0x15 0x85 0x1b 0x2a
+    0x01 0x01 0x01 0x01 0x00 0xff 0xff 0xff 0xff 0xff 0xff 0x00 0x15 0x85 0x1b
+    0x2a 0x01 0x01 0x01 0x01
+  checkBytesComma: 0x00,0xff,0xff,0xff,0xff,0xff,0xff,0x00,0x15,0x85,0x1b,0x2a,
+    0x01,0x01,0x01,0x01,0x00,0xff,0xff,0xff,0xff,0xff,0xff,0x00,0x15,0x85,0x1b,
+    0x2a,0x01,0x01,0x01,0x01,0x00,0xff,0xff,0xff,0xff,0xff,0xff,0x00,0x15,0x85,
+    0x1b,0x2a,0x01,0x01,0x01,0x01
+  checkBytesIncorrect: 00 ff ff ff ff ff ff 00 15 85 1b 2a 01 01 01 01 ff 20
```

Comment:
This last one is incorrectly formatted you must assume decimal in this case.

Hex is only supported via

- `0x00, 0x00`  - , Space
- `0x00 0x00` - Space
- `0x00,0x00` - Comma no space.

- 00 - Decimal
- ff - invalid decimal